### ISSUE TITLE
Java code completion

### ISSUE THREAD

**Issue by leedonet73** [bobbylight/RSTALanguageSupport#44]
"Calendar.getInstance()." in Java code. In the same case, the code is not complete ?

If you return an  Class of a static method, is it not possible to generate the class code complete?

**Comment by bobbylight** [comment]
No, this project does not yet support multiple levels of fields/getters with code completion (only 1). I'm closing this as a duplicate of #72.
